src/components/common/ConnectWalletButton/ChooseWalletModal/ChooseWalletModal.tsx

```diff
@@ -192,7 +145,24 @@ const ChooseWalletModal: React.FC<ChooseWalletModalProps> = ({
         <Trans>Select a wallet</Trans>
       </Modal.Title>
       <Modal.Content width={400}>
+        <ProtocolDisclaimer padding={2}>
+          <Flex col>
+            <Flex.Item>
+              <Trans>
+                By connecting a wallet, you agree to the{' '}
+                <a href="https://spectrum.fi/cookie-policy">Cookie policy</a>{' '}
+                and acknowledge that you have read and understand the{' '}
+                <a href="https://spectrum.fi/protocol-disclaimer">
+                  Protocol disclaimer
+                </a>
+                .
+              </Trans>
+            </Flex.Item>
+          </Flex>
+        </ProtocolDisclaimer>
```

Comment:
I'm not sure that it should be a custom component. But @Ridel1e knows better

Reply:
added a component folder for that.